src/main/java/uk/gov/mca/beacons/api/jpa/entities/Beacon.java

```diff
@@ -57,6 +57,12 @@
 
   private String csta;
 
+  private String beaconType;
```

Comment:
Wondering if (like further down for BeaconStatus) a BeaconTypes enum would be good - but at the same time, I guess that adds another place to make changes if the list of Beacon Types needed changing.